submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/security/web/SecurityResource.kt

```diff
@@ -45,6 +46,15 @@ class SecurityResource(
         securityService.registerUser(register)
     }
 
+    @PostMapping(value = ["/check-user"])
+    @ApiOperation("Check user registration")
```

Comment:
I think that if we're going with this approach, the documentation should be better something like:

```
Checks if a user with the given email is registered in the system. In case it's not, a new user will be created using such email and the given user name. The new user will be inactive and should be activated via activation link.
```